calicoctl/calicoctl.go

```diff
@@ -100,6 +101,20 @@ Description:
 		command := arguments["<command>"].(string)
 		args := append([]string{command}, arguments["<args>"].([]string)...)
 
+		// A "datastore migrate import" command should skip version mismatch checking,
+		// since the datastore will not have the cluster version set
+		isImport := strings.HasPrefix(strings.Join(args, " "), "datastore migrate import")
```

Comment:
hm... you're right, let me try that, thanks!

Reply:
Sorry for returning to this, but I think it will work that way as is. `args` is already created by joining `arguments["<command>"]` and `arguments["<args>"]` (which will contain the subcommand). So this will already have removed "top-level" calicoctl options (such as `-l debug`), and I don't think it's possible to have options between the command and subcommands due to the docopts parsing of the `datastore` command...

Reply:
Made the changes per our offline conversation, thanks for the improvement suggestions!